src/main/webapp/res/js/common.js

```diff
@@ -0,0 +1,17 @@
+const juniorElem = document.querySelector('.junior-class');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/koreait0000/NONG.4/pullRequest?prid=7788346)
```suggestion
const juniorElem = document.querySelector(".junior-class");
```